lib/engine/game/g_18_usa/step/acquire.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require_relative '../../g_1817/step/acquire'
+require_relative 'scrap_train_module'
+module Engine
+  module Game
+    module G18USA
+      module Step
+        class Acquire < G1817::Step::Acquire
+          include ScrapTrainModule
+          def actions(entity)
+            actions = super
+            if entity.corporation? && entity == @buyer && entity.trains.any? { |t| @game.pullman_train?(t) }
+              actions = %w[pass] if actions.empty?
+              actions << 'scrap_train'
+            end
+            actions
+          end
+
+          def pass_description
+            if @offer
+              'Pass (Offer for Sale)'
+            elsif @auctioning
+              'Pass (Bid)'
+            elsif @buyer && can_take_loan?(@buyer)
+              'Pass (Take Loan)'
+            elsif @buyer && can_payoff?(@buyer)
+              'Pass (On payoff Loan)'
+            elsif @buyer
+              'Pass (Scrap Train)'
+            end
```

Comment:
When you want to append functionality, its best if you call super and add your new logic around it. Try to keep the code DRY, so that rework and bugs are easier to resolve across the code base.